src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -121,22 +127,7 @@ public function validate($entity, Constraint $constraint)
             return;
         }
 
-        if (null !== $constraint->entityClass) {
```

Comment:
I guess there was a reason for this?

https://github.com/symfony/symfony/pull/15002

Reply:
well.. given $entity now can be any $object, it doesnt make sense to validate the inheritance anymore. I dont think it hurts.

cc @ogizanagi